services/messaging/pom.xml

```diff
@@ -29,6 +29,10 @@
       <groupId>org.eclipse.hono</groupId>
       <artifactId>hono-service-base</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-tcnative</artifactId>
```

Comment:
it should be sufficient to add this to `services/pom.xml` instead of each service individually ...